kitsune/sumo/static/sumo/js/tests/form-wizard-configure-step-tests.js

```diff
@@ -0,0 +1,82 @@
+import { expect } from "chai";
+import { ConfigureStep } from "sumo/js/form-wizard-configure-step";
+
+async function waitForEvent(target, eventName) {
+  return new Promise(resolve => {
+    target.addEventListener(eventName, resolve, { once: true });
+  })
+}
+
+describe("configure-step custom element", () => {
+  let step;
+  let slot;
+
+  before(() => {
+    customElements.define("configure-step", ConfigureStep);
```

Comment:
do we want to just do the `customElements.define` once in `form-wizard-configure-step.js` instead of both here and in `device-migration-wizard.js`? I think we could then this in `device-migration-wizard.js`:

```js
import "sumo/js/form-wizard-configure-step";
```

and everything would just work™️ 

curious if there's another reason for delaying the `customElements.define` call

Reply:
Ah, yes, I can do that. I just recall us doing things in Firefox-land where we try to avoid side-effects from imports: https://searchfox.org/mozilla-central/rev/45b20d00311bc2170dfbdae8a91bc4dca29717d8/browser/components/storybook/stories/migration-wizard.stories.mjs#7

but that's all just old reflex. What you're suggesting makes sense.